src/EnergyPlus/GroundTemperatureModeling/SiteFCFactorMethodGroundTemperatures.cc

```diff
@@ -69,21 +69,12 @@ std::shared_ptr<SiteFCFactorMethodGroundTemps> SiteFCFactorMethodGroundTemps::FC
     // SUBROUTINE INFORMATION:
     //       AUTHOR         Matt Mitchell
     //       DATE WRITTEN   Summer 2015
-    //       MODIFIED       na
-    //       RE-ENGINEERED  na
 
     // PURPOSE OF THIS SUBROUTINE:
     // Reads input and creates instance of Site:GroundTemperature:FCfactorMethod object
 
-    // USE STATEMENTS:
-    using namespace GroundTemperatureManager;
-
-    // Locals
     // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
     bool found = false;
-    int NumNums;
-    int NumAlphas;
-    int IOStat;
```

Comment:
[src/EnergyPlus/GroundTemperatureModeling/SiteFCFactorMethodGroundTemperatures.cc:84]:(style),[variableScope],The scope of the variable 'NumNums' can be reduced.
[src/EnergyPlus/GroundTemperatureModeling/SiteFCFactorMethodGroundTemperatures.cc:85]:(style),[variableScope],The scope of the variable 'NumAlphas' can be reduced.
[src/EnergyPlus/GroundTemperatureModeling/SiteFCFactorMethodGroundTemperatures.cc:86]:(style),[variableScope],The scope of the variable 'IOStat' can be reduced.